### PR TITLE
Add version references for Commander - Closes #111

### LINKED ISSUE #111: Add links to older docs version to introduction pages
At the moment we display the latest version of our documentation for the different products on lisk.io.

We should add references to older versions on the introduction page for each product.

- [x] core
- [x] commander
- [x] elements

### PR DESCRIPTION
**Pull request by Tschakki**
Closes #111

### CODE REVIEW

File: introduction.md [hunk 1]
```diff
@@ -12,12 +22,16 @@
...
-This will facilitate delegates, node operators, contributors to the Lisk codebase, developers using Lisk’s infrastructure (and anyone else who’s interested) to make use of Lisk’s functionality from the command line with ease.
+## Versions
 
-Lisk Commander allows you to communicate with a remote or local node and carry out Lisk-related functionality.
-Going forward we’ll be adding more commands to provide a comprehensive suite for all your Lisk-related needs.
+We release new versions of Lisk Commander regularly. Documentation on [lisk.io](https://lisk.io/documentation) will keep up to date with version updates. The table below provides an overview about the former releases and documentation versions of Lisk Commander.
+
+Version | Release date <br> (yy/mm/dd)| Documentation reference
+---     | ---         | ---
+[v2.2.0](https://github.com/LiskHQ/lisk/releases/tag/v2.1.0) | 19/0x/xx | *Current version, live on lisk.io/documentation*
```
Comment (shuse2) [inline introduction.md:36]: do we need this `2.2.0` link now?

File: introduction.md [hunk 2]
```diff
@@ -1,7 +1,17 @@
 # Lisk Commander Documentation
 
+Lisk Commander aims to be a low entry point and the most convenient tool for developers who are looking for an efficient way of coding with a command line interface (CLI).
+With Lisk Commander, it is easier and quicker to plug new commands into the CLI than building a Graphical User Interface (GUI) around every command.
```
Comment (shuse2) [inline introduction.md]: these two sentence sounds like more of why create the lisk-commander, but not for external communication.

maybe it's good to communicate what exactly commander is right now?

**Review by shuse2 — CHANGES_REQUESTED** [review]

File: introduction.md (hunk 1 above)
Comment (Tschakki) [reply introduction.md:36]: yes, as this PR will be merged into the v2.2.0 docs release for Lisk Elements. The exact release date will be added later when it's set.

File: introduction.md (hunk 2 above)
Comment (Tschakki) [reply introduction.md]: I added suggestion for a new introduction text, feedback welcome! :)

**Review by shuse2 — APPROVED** [review]

**Pull request merged**